app/src/main/java/org/simple/clinic/enterotp/EnterOtpEffectHandler.kt

```diff
@@ -45,6 +45,7 @@ class EnterOtpEffectHandler @AssistedInject constructor(
         .addAction(ShowNetworkError::class.java, uiActions::showNetworkError, schedulers.ui())
         .addAction(ShowUnexpectedError::class.java, uiActions::showUnexpectedError, schedulers.ui())
         .addTransformer(LoadOtpEntryProtectedStates::class.java, loadOtpEntryStates())
+        .addAction(ResetOtpAttemptLimit::class.java, { bruteForceProtection.resetFailedAttempts() }, schedulers.io())
```

Comment:
Just a personal preference, we can use method reference here.

```suggestion
        .addAction(ResetOtpAttemptLimit::class.java, bruteForceProtection::resetFailedAttempts, schedulers.io())
```